heron/scheduler-core/src/java/com/twitter/heron/scheduler/dryrun/SubmitDryRunResponse.java

```diff
@@ -0,0 +1,32 @@
+// Copyright 2016 Twitter. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.twitter.heron.scheduler.dryrun;
+
+import com.twitter.heron.api.generated.TopologyAPI;
+import com.twitter.heron.spi.common.Config;
+import com.twitter.heron.spi.packing.PackingPlan;
+
+/**
+ * Dry-run response, which extends RuntimeException because it is easy
+ * to propagate up
+ */
+public class SubmitDryRunResponse extends DryRunResponse {
```

Comment:
The reason of doing so is that later in #1622 (AbstractMain), I'd like to have a common abstract method like `protected abstract String renderDryRunResponse(DryRunResponse)` in `AbstractMain`. 

`SubmitterMain` will then override the abstract with a method `String renderDryRunResponse(SubmitDryRunResponse)` and `RuntimeManagerMain` will have `String renderDryRunResponse(UpdateDryRunResponse)`.

It is of course still OK to follow your suggestion, but I'd like to see naming here look more consistent (SubmitterMain renders `SubmitDryRunResponse` instead of `DryRunResponse`).